src/Microsoft.CodeQuality.Analyzers/VisualBasic/Maintainability/BasicUseNameofInPlaceOfString.vb

```diff
@@ -0,0 +1,22 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.VisualBasic
+Imports Microsoft.CodeQuality.Analyzers.Maintainability
+
+Namespace Microsoft.CodeQuality.VisualBasic.Analyzers.Maintainability
+
+    ''' <summary>
+    ''' CA1507: Use nameof to express symbol names
+    ''' </summary>
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Public NotInheritable Class BasicUseNameofInPlaceOfStringAnalyzer
```

Comment:
Dumb question: do we call these "Basic" or "VisualBasic"? Does the name show up anywhere?

Reply:
This name is consistent with the other analyzers in Roslyn-Analyzers repo.  I don't believe the name is visible anywhere, but @mavasani would know for sure.